src/transactions/functions.rs

```diff
@@ -106,5 +148,153 @@ mod tests {
     }
 }
 
-
-
+fn build_query(public_key: Option<String>) -> transactions_query::TransactionQueryInput {
```

Comment:
This is unfortunate. I tried following a builder pattern or trying 

```
transactions_query::TransactionQueryInput {
  // my fields
  ..Default::defaults()
}
```

But the model is not under my control and I couldn't seem apply the Default macro

Reply:
Can you omit them?

```
transactions_query::TransactionQueryInput {
  from: public_key,
  canonical: Some(true),
}
```

Reply:
No, unfortunately. Struct not under my control